### ISSUE TITLE
questions about trajectories,network

### ISSUE THREAD

**Issue by zttt04** [DigiRL-agent/digirl#52]
Hi, I am fascinated by your greaaaaat work!Now I am following your work but encountering some questions,could you please help me solve them?
1.I am doubted about the trajectories source. I don't know how you turn AitW into format like this 

![Image](https://github.com/user-attachments/assets/0520a884-93de-48b4-ba10-9573317970a8).
Could you please provide the processing python files?

Also I noticed that 

'image_path': '/home/yifeizhou/logs/general-off2on-zeroshot-data-2/images/test1/1715169167.4070764_1.png', 

I wonder that if I can't get the image can I use the trajectories to train? Or the image information has already been extract to the image_features so that we don't need the origin image anymore.

2.When I tried to evaluate your checkpoint, it struct at this step.I guess this is the network problem(I live in cn,so many websites I can't open),and then I add my host proxy to the emulator but it still fail to search. So is there any useful methods that I can solve
Finally, congratulations on your great work! Have a good day!

![Image](https://github.com/user-attachments/assets/14596fdd-bc5d-4e3a-9490-6eb5cf29dfaf)

3.Due to the network problem,I install EDGE and modify the test set.For example:open Edge;Open Edge and use edge to search information about Peking University. It can easily finish the first task,however, when it occur to the second, it quit Edge and turn to Chrome to search the information. And from then on I pay attention to the way it open app,and finally find out it turn on app by using chrom too. And I think it may because of the training data,from which many task needs to turn on chrom to search or something else.

Finally, congratulations on your great work! Have a good day!
